setup.py

```diff
@@ -14,6 +14,7 @@
     'apscheduler',
     'icalendar',
     'pytz',
+    'googleapiclient'
```

Comment:
can you double check this is the correct package name? iirc it was something like `google-api-python-client`